addons/event/models/event_ticket.py

```diff
@@ -18,11 +18,6 @@ class EventTemplateTicket(models.Model):
         help="A description of the ticket that you want to communicate to your customers.")
     event_type_id = fields.Many2one(
         'event.type', string='Event Category', ondelete='cascade', required=True)
-    # sale
-    start_sale_date = fields.Date(string="Registration Start")
-    end_sale_date = fields.Date(string="Registration End")
-    is_expired = fields.Boolean(string='Is Expired', compute='_compute_is_expired')
-    sale_available = fields.Boolean(string='Is Available', compute='_compute_sale_available')
```

Comment:
beh ! you play with my feet little hobbit ! :open_mouth: 
Why not in original commit ?

Reply:
Meh. I lessened commit number and merged some of them. There is still some mismatch but first commit keeps somehow previous behavior, next commit ref a bit models after some discussions with LNA.